backend/app/adapter/graphql/resolver/authquery.go

```diff
@@ -40,16 +41,21 @@ func (v AuthQuery) URL(args *URLArgs) (*URL, error) {
 
 // ChangeLog retrieves full ChangeLog from persistent storage
 func (v AuthQuery) ChangeLog() (ChangeLog, error) {
+	user, err := viewer(v.authToken, v.authenticator)
```

Comment:
```suggestion
	_, err := viewer(v.authToken, v.authenticator)
```

Let's do this for now to avoid having to change the interface for `GetLastViewedAt`.